pkg/tests/postman/collection_data.json

```diff
@@ -0,0 +1,7 @@
+[
```

Comment:
any reason to add `_data` to each file?

Reply:
Any object of the JSON array is an iteration the postman going to do with this API, if a new platform starts to implement staking, we need to update the `staking_data.json` file

https://learning.postman.com/docs/postman/collec
tion-runs/running-multiple-iterations/